### PR TITLE
Update to 2.4

### PR DESCRIPTION
**Pull request by jayfurmanek**
Draft for update to TB 2.4

### CODE REVIEW

**Comment by npanpaliya** [comment]
1. Build number is to be incremented. 
2. 0003-Disable-tensorboard.dev-uploader.patch needs to be updated as we are adding google-auth and google-auth-oauthlib.

**Comment by jayfurmanek** [comment]
Build number will still be 1 since we've updated the version.
I've enabled dev uploader here. We can discuss if needed.

**Review by npanpaliya — APPROVED** [review]

**Comment by npanpaliya** [comment]
> Build number will still be 1 since we've updated the version.
> I've enabled dev uploader here. We can discuss if needed.

No, that's okay. We also need to delete that patch file.

**Review by npanpaliya — APPROVED** [review]

File: buildscripts/set_python_path_for_bazelrc.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# *****************************************************************
+#
+# Licensed Materials - Property of IBM
+#
+# (C) Copyright IBM Corp. 2018, 2019. All Rights Reserved.
```
Comment (npanpaliya) [inline buildscripts/set_python_path_for_bazelrc.sh:6]: This needs to be updated.

**Pull request merged**